label_studio/annotation_templates/generative-ai/llm-ranker/config.yml

```
@@ -22,24 +22,32 @@ config: |
         <List name="answers" value="$items" title="All Results" />
         <Style>
           .htx-ranker-column {
-            background: #f8f8f8;
+            background: var(--color-neutral-background);
             width: 50%;
-            padding: 20px;
-            border-radius: 3px;
-            box-shadow: 0px 2px 5px 0px rgba(0,0,0,0.1);
+            padding: var(--spacing-tight);
+            border-radius: var(--corner-radius-smaller);
+            box-shadow: 0px 2px 5px 0px rgba(var(--color-neutral-shadow-raw) / 10%);
+            border: 1px solid var(--color-neutral-border);
           }
       
           .htx-ranker-item {
-            background: #e0e0e0;
-            color: #333;
+            background: var(--color-neutral-surface);
+            color: var(--color-neutral-content);
             font-size: 16px;
             width: 100%;
-            padding: 10px;
-            margin-bottom: 10px;
-            border-radius: 3px;
-            box-shadow: 0px 2px 5px 0px rgba(0,0,0,0.1);
+            padding: var(--spacing-tight);
+            margin-bottom: var(--spacing-tight);
+            border-radius: var(--corner-radius-smaller);
+            box-shadow: 0px 2px 5px 0px rgba(var(--color-neutral-shadow-raw) / 10%);
+            border: 1px solid var(--color-neutral-border);
           }
-          .htx-ranker-item p:last-child { display: none }
+          
+          
+          .htx-ranker-item:hover {
+            background: var(--color-neutral-surface-hover);
+          }
+          
+          .htx-ranker-item p:last-child { display: none };
```

I'm very curious what's the element we are trying to hide here, but we had this style already... at least let's remove semicolon, so it stays unchanged
```suggestion
          .htx-ranker-item p:last-child { display: none }
```